landavailability/api/management/commands/import_addresses.py

```diff
@@ -0,0 +1,41 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.gis.geos import Point
+from api.models import Address
+import csv
+
+
+class Command(BaseCommand):
+    help = 'Import addresses from a CSV file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('csv_file', type=str)
+
+    def handle(self, *args, **options):
+        csv_file_name = options.get('csv_file')
+
+        if csv_file_name:
+            with open(csv_file_name, newline='') as csvfile:
```

Comment:
As you're going to be doing this a lot, you could possibly abstract this out into a CSVImportCommand that extends BaseCommand and just calls a method called process_row(row) ?  This way future importers can forget about the boiler-plate, although you may need to make a call to super.__init__() if you need a different delimiter (for instance).

It's okay for now, but perhaps a refactor when you have more than one CSV Import?